crates/matrix-sdk-crypto/src/session_manager/group_sessions/share_strategy.rs

```diff
@@ -42,6 +46,21 @@ pub enum CollectStrategy {
         ///       trusted via interactive verification.
         ///     - It is the current own device of the user.
         only_allow_trusted_devices: bool,
+        /// If true, when a verified user has an unsigned device the key sharing
+        /// will fail with an error. If false the key will be
+        /// distributed to that unsigned device. In order to resolve
```

Comment:
```suggestion
        /// If `true`, and a verified user has an unsigned device, key sharing
        /// will fail with an error. Otherwise, keys are shared with
        /// unsigned devices as normal. 
        ///
        /// In order to resolve
```